posthog/hogql_queries/experiments/test/experiment_query_runner/__snapshots__/test_base.ambr

```diff
@@ -54,7 +54,7 @@
               exposures.entity_id) AS metric_events
   GROUP BY metric_events.variant
   LIMIT 100 SETTINGS readonly=2,
-                     max_execution_time=60,
+                     max_execution_time=180,
```

Comment:
Nitpick: moving these fixed values (like max_ast_elements and team id overrides) to constants. We repeat them all the time 🤓

Reply:
These are just snapshots of the compiled queries and not something we type by hand, so not sure if I understand what you mean here?

Reply:
ignore this, I commented before looking at the `HogQLGlobalSettings` addition.

Reply:
Ah, I see 👍